### PR TITLE
[Azure firewall manager virtual hub] - update api versions and create…

### PR DESCRIPTION
**Pull request by mumian**
… bicep

# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@vhorne - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@vhorne - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.network/fwm-docs-qs/main.bicep [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+        ]
+        ids: [
+          {
+            id: hubRouteTable.id
+          }
+        ]
+      }
+    }
+  }
+  dependsOn: [
```
Comment (bmoore-msft) [inline quickstarts/microsoft.network/fwm-docs-qs/main.bicep:61]: move this above properties

File: quickstarts/microsoft.network/fwm-docs-qs/main.bicep [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+  name: 'Jump-SN'
+  properties: {
+    addressPrefix: '10.0.2.0/24'
+    routeTable: {
+      id: routeTable.id
+    }
+    privateEndpointNetworkPolicies: 'Enabled'
+    privateLinkServiceNetworkPolicies: 'Enabled'
+  }
+  dependsOn: [
```
Comment (bmoore-msft) [inline quickstarts/microsoft.network/fwm-docs-qs/main.bicep:171]: move this above properties

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**